mkdocs.yml

```diff
@@ -97,6 +97,8 @@ nav:
           - Parallel Processing Python: _user_guide/best_practices_parallel_processing_python.md
       - R:
           - Best Practices in R: _user_guide/best_practices_r.md
+      - Matlab:
+          - Matlab Slurm Jobs:  _user_guide/best_practices_matlab_slurm_job.md
```

Comment:
Change "Matlab Slurm Jobs" to "Slurm Jobs" since "Matlab" is already the title of the main section